snippets/snip-private-clusters-public-ingress.adoc

```diff
@@ -0,0 +1,12 @@
+// Text snippet included in the following modules:
+//
+// * modules/private-clusters-default.adoc
+// * modules/private-clusters-about.adoc
+// * modules/private-clusters-about-aws.adoc
+
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+LoadBalancer-type services created by administrating users are publicly accessible unless they are explicitly annotated as private.
```

Comment:
I think this is only true if the cluster has public subnets.  As mentioned in the AWS and GCP documentation, public subnets are _not required_ to be configured, but they _can_ be configured, and that's when services can be accidentally made public.
```suggestion
If the cluster has any public subnets, LoadBalancer-type services created by administrating users may be publicly accessible unless they are explicitly annotated as private.
```

Reply:
Hey @skrthomas this might just be me not understanding the problem, but it is unclear to me what you mean by "LoadBalancer-type services created by administrating users."

Reply:
@mjpytlak LoadBalancer-type services created by admin users vs LoadBalancer services created by the installer. Its my understanding that the later would be private in private installs. The of a public LoadBalancer service in a private install is only with user-created (usually admins).